api/container/containerv2/clusters.go

```diff
@@ -30,6 +30,7 @@ type ClusterCreateRequest struct {
 	DefaultWorkerPoolEntitlement string           `json:"defaultWorkerPoolEntitlement"`
 	CosInstanceCRN               string           `json:"cosInstanceCRN"`
 	WorkerPools                  WorkerPoolConfig `json:"workerPool"`
+	SecurityGroupIDs	         []string	`json:"securityGroupIDs"`
```

Comment:
should it always be included to the request? or should we add `omitempty` to it?

Reply:
thank you, I'll add the `omitepmty`